tasks/build-exports-list.js

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import path from 'path';
+import fs from 'fs';
+import fg from 'fast-glob';
+
+const build = (packageJSONPath) => {
+    const exportsPath = path.join(packageJSONPath, '..', 'exports.json');
+    if (fs.existsSync(exportsPath)) {
+        return;
+    }
+    const packageJSON = JSON.parse(fs.readFileSync(packageJSONPath, 'utf8'));
+    const exports = packageJSON.exports;
+    if (!exports) {
+        return;
+    }
+    delete exports['.'];
+    delete exports['./package.json'];
+    fs.writeFileSync(exportsPath, JSON.stringify(exports, null, '    '), {
```

Comment:
It can be edited, which is why we don't write it if it already exists. While _most_ packages won't need to do so, not being able to would prevent packages like [styles](https://github.com/adobe/spectrum-web-components/pull/2257/files#diff-04e371540a2335aafa408bf506165a4305a795efcda81bd5c604aaaf0c5534a0) or [theme](https://github.com/adobe/spectrum-web-components/pull/2257/files#diff-1d44a5f253c534f58a60f8319b939b023cf07622ea7763ef42757223e25a1cd4) from being able to build a complete exports map.